definitions/features/foreman_database.rb

```diff
@@ -16,7 +16,15 @@ def configuration
   end
 
   def config_files
-    [postgresql_conf]
+    [postgresql_conf] if el?
```

Comment:
For a guard clause to work, this needs to be an explicit return I think.

```suggestion
    return [postgresql_conf] if el?
```

Reply:
You could also use a ternary operator:

```suggestion
    el? ? [postgresql_conf] : []
```